app/views/layouts/application.html.erb

```diff
@@ -5,6 +5,7 @@
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
 
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css" integrity="sha384-9aIt2nRpC12Uk9gS9baDl411NQApFmC26EwAOH8WgZl5MYYxFfc+NcPb1dKGj7Sk" crossorigin="anonymous">
```

Comment:
For future changes, rails has some great asset tag helpers that can make use of embedded ruby here. Not necessary, but can help cut down on writing raw markup like this. https://apidock.com/rails/ActionView/Helpers/AssetTagHelper